docs/developer/order-expiration.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Order Expiration
+---
+
+:::info
+
+expireOrdersAfter was introduced in **Saleor 3.13**
+
+deleteExpiredOrdersAfter was introduced in **Saleor 3.14**
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+## Introduction
+
+Saleor allows finalizing checkout without payment. This can be achieved in two ways:
+
+- by setting `orderSettings.allowUnpaidOrders` for a given `Channel` to `true` and calling `checkoutComplete` without any payment/transaction object
+- by using the `orderCreateFromCheckout` mutation with first order then payment flow.
+
+If the payment process is canceled by the customer and the order remains unpaid for some time, it is possible to set automatic stock releases and remove the order.
+
+## Configuration
+
+To set the expiration of orders, specify the number of minutes after which the orders should expire by setting `expireOrdersAfter`.
+Additionally, the `deleteExpiredOrdersAfter` value can be modified. By default, this is set to 60 days, but it can be changed to anywhere between 1 and 120 days.
+
+```graphql
+mutation ChannelUpdate {
+  channelUpdate(
+    id: "Q2hhbm5lbDoyMjQz"
+    input: {
+      orderSettings: { expireOrdersAfter: 360, deleteExpiredOrdersAfter: 30 }
+    }
+  ) {
+    channel {
+      id
+      orderSettings {
+        expireOrdersAfter
+        deleteExpiredOrdersAfter
+      }
+    }
+    errors {
+      field
+      code
+      message
+    }
+  }
+}
+```
+
+```json
+{
+  "data": {
+    "channelUpdate": {
+      "channel": {
+        "id": "Q2hhbm5lbDoyMjQz",
+        "orderSettings": {
+          "expireOrdersAfter": 360,
+          "deleteExpiredOrdersAfter": 30
+        }
+      },
+      "errors": []
+    }
+  },
+  "extensions": {
+    "cost": {
+      "requestedQueryCost": 0,
+      "maximumAvailable": 50000
+    }
+  }
+}
+```
```

Comment:
You could add here a sentence or two that will explain that as a result of this example, your unpaid orders will expire after 360 minutes, and will be deleted after 30 days.